packages/react-relay/__tests__/__generated__/LiveResolversTestCounterContextQuery.graphql.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @oncall relay
+ *
+ * @generated SignedSource<<0b678b2baa677776abf9ddcfd58c80ec>>
+ * @flow
+ * @lightSyntaxTransform
+ * @nogrep
+ */
+
+/* eslint-disable */
+
+'use strict';
+
+/*::
+import type { ClientRequest, ClientQuery } from 'relay-runtime';
+import type { LiveState } from "relay-runtime";
+import {counter_context as queryCounterContextResolverType} from "../../../relay-runtime/store/__tests__/resolvers/LiveCounterContextResolver.js";
+import type { LiveResolverContextType } from "../../../relay-runtime/mutations/__tests__/LiveResolverContextType";
+// Type assertion validating that `queryCounterContextResolverType` resolver is correctly implemented.
+// A type error here indicates that the type signature of the resolver module is incorrect.
```

Comment:
For those folks that check the artefacts into SCM, I wonder if this will take up too much space vs the benefit of the inline documentation?

Reply:
FWIW, the assertion and these two comment lines predate our changes.